package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "eslint --ext .js,.vue src",
     "lint:fix": "eslint --ext .js,.vue src --fix",
     "stylelint": "stylelint **/*.css **/*.scss **/*.vue",
-    "stylelint:fix": "stylelint **/*.css **/*.scss **/*.vue --fix"
+    "stylelint:fix": "stylelint **/*.css **/*.scss **/*.vue --fix",
+    "test": "echo 'TODO'"
```

Comment:
This probably shouldn't be here

Reply:
I'm pretty sure that this needed to be here for the CI checks to pass: https://github.com/nextcloud/news/runs/6544192521?check_suite_focus=true

in another PR we can try to add mocha/karma unit tests that can be run here, but for now it is just a TODO

Reply:
Or do you mean we should leave this ci check failing until they are properly implemented?

Reply:
Well, if it's not doing anything CI shouldn't be checking it iyam.

Reply:
Works for me